airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/ImageCell.tsx

```diff
@@ -18,6 +18,10 @@ const Link = styled.a`
 `;
 
 const ImageCell: React.FC<ImageCellProps> = ({ imageName, link }) => {
+  if (!link || !link.length) {
+    return <>{imageName}</>;
```

Comment:
Should we wrap the `imageName` in `<Text>`? Having issues uploading a custom connector locally to test this, but I believe we don't want raw text inside the `<td>`

```suggestion
    return <Text>{imageName}</Text>;
```

Reply:
I rewrote this component to use the `<Text/>` component and get the styled components out of there in https://github.com/airbytehq/airbyte/pull/21034/commits/321b68f80f32a1cf8690f72abce29371314b07d6

Reply:
Nice!